lib/phoenix_html/link.ex

```diff
@@ -188,6 +188,11 @@ defmodule Phoenix.HTML.Link do
 
   All other options are forwarded to the underlying button input.
 
+  When the `:method` is set to `:get` and the `:to` URL contains query
```

Comment:
I believe this happens regardless of the `:method`, no? Or is it only for `:get` indeed?

Reply:
I added some tests for post and delete [here](https://github.com/mattludwigs/query_string_test/commit/8b94f26a395b6736409639b960ec43fd71de4b41) and I am getting the query params from those, so it appears that the params are only striped for `:get`, unless I am miss understanding or my test is making a wrong assumptions.